src/hooks/useContract.ts

```diff
@@ -48,7 +50,7 @@ export function useTokenContract(tokenAddress?: string, withSignerIfPossible?: b
 
 export function useWETHContract(withSignerIfPossible?: boolean): Contract | null {
   const { chainId } = useActiveWeb3React()
-  return useContract(chainId ? WETH[chainId].address : undefined, WETH_ABI, withSignerIfPossible)
+  return useContract(chainId ? WETH[chainId]?.address : undefined, WETH_ABI, withSignerIfPossible)
```

Comment:
that' WETH contract for every network? where it's used for?

Reply:
yes it's a WETH contract for every network, it was throwing an error on the swap page when you are on the bsc chain, since the object doesn't have a value for bsc chain.